components/instantly/sources/new-event-instant/new-event-instant.mjs

```diff
@@ -0,0 +1,67 @@
+import instantly from "../../instantly.app.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  key: "instantly-new-event-instant",
+  name: "New Event in Instantly (Instant)",
+  description: "Emit new event when an activity occurs in your Instantly workspace.",
```

Comment:
Can we add links to the docs regarding webhooks?

Reply:
@GTFalcao. They don't have documentation for webhooks